GliaWidgets/SecureConversations/Confirmation/SecureConversations.ConfirmationViewModel.swift

```diff
@@ -53,11 +53,16 @@ extension SecureConversations.ConfirmationViewModel {
         backButtonCmd: Cmd,
         closeButtonCmd: Cmd
     ) -> Header.Props {
-        Header.Props(
+        var backButton: HeaderButton.Props?
+        if let backButtonStyle = style.header.backButton {
```

Comment:
This approach with `if let` works well, but maybe it makes sense to take advantage of `Optional.map` to do unwrapping and assignment in one take?
```
var backButton: HeaderButton.Props? = style.header.backButton.map { backButtonStyle in .init(tap: backButtonCmd, style: backButtonStyle) } 
```

Reply:
we introduced `unwrap` function also, perhaps it can be used also.

Reply:
Fix 8b6c42659f6ac84f0cb4f9f8cf8e4335fed6cc05